### ISSUE TITLE
Wordpress: Uncaught ReferenceError: Invalid left-hand side in assignment

### ISSUE THREAD

**Issue by kit-johnson** [michalsnik/aos#499]
## This is:

- question

## Specifications

  - AOS version: 2.3.4
  - OS: Windows 10
  - Browser: Chrome 75

## Actual Behavior

When I load AOS from within Wordpress, I get an error in the browser console: `Uncaught ReferenceError: Invalid left-hand side in assignment`. No animations work.

## Steps to Reproduce the Problem

1. Direct download [latest version](https://github.com/michalsnik/aos/archive/master.zip)
2. Copy `aos.js` and `aos.css` from `aos-master\dist` folder
3. Enqueue these files using Wordpress `wp_enqueue_script`
4. Load page and see the console error `Uncaught ReferenceError: Invalid left-hand side in assignment`. No animations work.

## Detailed Description

First I tried installing with npm and found this error. I checked a couple of past issues and thought it might be an issue with my local install. Downloading from the direct link, however, and the problem still remains.

If I load aos from a plain html file (not within Wordpress), everything is working fine, so the problem is Wordpress. I don't know if this is fixable or if I should give up.

**Comment by adamskaiza** [comment]
Hi, did you solve this problem?

**Comment by kit-johnson** [comment]
@adamskaiza Yes, indirectly, I installed a plugin from the Wordpress repository called _Animate It!_ It introduces unwanted behaviour (such as a new button in any rich text editor field), but otherwise I'm happy with it.
